UIKit_DZ/UIKit_DZ/MenuItemConfigScreen/MenuItemCofigurationViewController.swift

```diff
@@ -3,5 +3,176 @@
 
 import UIKit
 
-// Vaka
-final class MenuItemCofigurationViewController: UIViewController {}
+/// Экран с конфигарцией напитка
+final class MenuItemCofigurationViewController: UIViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let modifireLabel = "Модификация"
+        static let promoCode = "Лови промокод roadmaplove на 50 отжиманий"
+        static let menuItems = ["Американо", "Капучино", "Латте"]
+    }
+
+    // MARK: - visual components
+
+    /// кастомная верхня белая вьюшка
+    private lazy var topCustomView: UIView = {
+        let customView = UIView()
+        customView.frame = .init(x: 0, y: 0, width: view.frame.width, height: 346)
+        customView.backgroundColor = #colorLiteral(red: 0.9428322315, green: 0.8812621236, blue: 0.8162202239, alpha: 1)
+        customView.layer.cornerRadius = 20
+        customView.layer.maskedCorners = [.layerMinXMaxYCorner, .layerMaxXMaxYCorner]
+        return customView
+    }()
+
+    /// круглая вьюшка для кнопки назад
+    private lazy var backButtonBackgroundView: UIView = {
+        let customView = UIView()
+        customView.frame = .init(x: 7, y: 50, width: 44, height: 44)
+        customView.backgroundColor = #colorLiteral(red: 0.9217587709, green: 0.9660443664, blue: 0.969683826, alpha: 1)
+        customView.layer.cornerRadius = 20
+        return customView
+    }()
+
+    /// изображение с выбором кофе
+    private lazy var itemImage: UIImageView = {
+        let product = UIImageView()
+        product.frame = .init(x: 0, y: 0, width: 150, height: 150)
+        product.center.x = topCustomView.center.x
+        product.center.y = topCustomView.center.y
+        product.image = UIImage(named: "americano")
+        return product
+    }()
+
+    /// массив сс картинкамми для сегмента
+    private lazy var imageArray = [
```

Comment:
```suggestion
    private lazy var imageArray = [
```
не оч хорошая практика нужно массив coffeNames только имен